sandbox.h

```diff
@@ -1,5 +1,7 @@
-#include_next <sandbox.h>
+// #include_next <sandbox.h>
```

Comment:
We should not include commented-out code in this way. Should this be reverted?

Reply:
I believe it resulted in an import loop, but can try reverting.

Reply:
Can confirm it appears to result in an import loop (see [here](https://github.com/L1ghtmann/headers/actions/runs/6921510990/job/18827190138))

Reply:
hm. Seems like it was working before this PR. I wonder if it has something to do with "compiling" these headers directly instead of importing them into an implementation file first.
I'm hesitant to make this change

Reply:
Perhaps just a simple wrap to handle both cases, as that's likely most appropriate anyway
```c
#if __has_include_next(<sandbox.h>)
#   include_next <sandbox.h>
#endif
```

Reply:
Still [fails](https://github.com/L1ghtmann/headers/actions/runs/6921581220/job/18827349729), unfortunately

Reply:
[Reverted](https://github.com/theos/headers/pull/89/commits/605ed81dfb605720c6eae55dfe49dcfb0ac679b4) change to 'compile' headers directly.